### PR TITLE
Hotfix: Fix the events endpoint not working

### PR DESCRIPTION
**Pull request by AydanPirani**
IDK what I changed, but I guess the /event endpoint is working now???

looks like an issue with vercel deployment?

### CODE REVIEW

**Review by lasyaneti — COMMENTED** [review]
both this pr and the last pr work fine in the preview. can we test prod by deploying branch on prod instead of merging to main every time

**Comment by AydanPirani** [comment]
> both this pr and the last pr work fine in the preview.
 
Acknowledged, I didn't test much in the previews. However, in the previous PR, while it worked fine on the preview deployment (taking your word for it), the events endpoint didn't work in prod, but it returned an empty array instead. This PR resolves that issue - although nothing changed significantly, the new push to prod allowed us to reset the environment variables on Vercel, hence allowing for the deployment to work as expected.

>can we test prod by deploying branch on prod instead of merging to main every time

Completely agree. In fact, I've been deploying branch on prod, and then merging in hotfixes once the issue has been resolved. This way, I can make incremental changes without cluttering main, but we can also merge in the changes once the issue has been resolved - I'm sure we both agree on the importance of merging in a hotfix ASAP, after issue resolution. Note that the current prod deployment is running off the current hotfix branch (action link: https://github.com/HackIllinois/adonix/actions/runs/6294058495).

@lasyaneti  Please acknowledge comment and approve PR :)

**Comment by AydanPirani** [comment]
Also, @lasyaneti  one more thing to note - this is generally one of those implicit things, but I just wanted to clarify that pushing to main effectively does nothing when it comes to deploying to prod. This happens only when we **CREATE A RELEASE**, which can be done off any branch (i.e. this current hotfix one). 

HOWEVER, if branches aren't reconciled soon enough, this leads to the question of "which hotfix is the right prod one?", and hence we should try to merge deployment branches as soon as possible. For example, if we had another issue pop up right now, would we make changes on this branch (that's now production)? Do we make a new hotfix branch and merge that branch into this branch? It's very very easy to mess things up that way, so I'm def leaning towards ASAP hotfix merges

File: src/services/event/event-formats.ts [hunk 1]
```diff
@@ -26,10 +27,13 @@ export interface EventFormat {
...
  */
 export function isEventFormat(obj: EventFormat): boolean {
+	if (typeof obj.id !== "string" || obj.id.length != Constants.EVENT_ID_LENGTH) {
+		return false;
+	}
+
 	if (
 		typeof obj.isPrivate !== "boolean" ||
 		typeof obj.displayOnStaffCheckIn !== "boolean" ||
-		typeof obj.id !== "string" ||
```
Comment (lasyaneti) [inline src/services/event/event-formats.ts:32]: 😭

**Review by lasyaneti — APPROVED** [review]

**Pull request merged**